stdlib_shims.opam

```diff
@@ -9,4 +9,4 @@ tags: ["stdlib" "compatibility" "org:ocaml"]
 license: ["typeof OCaml system"]
 available: [ ocaml-version >= "4.02.3" ]
 depends: ["dune"]
-build: [ "dune" "build" "-p" name ]
+build: [ "dune" "build" "-p" name "-j" jobs ]
```

Comment:
By the way I think you should open a discussion on `opam` about the use of this variable. This variable is supposed to control the *inter-package* parallelism used by `opam` itself. I think this should be distinguished from the parallelism you want to use in the build system of packages.

Reply:
We had a discussion about this here: https://github.com/ocaml/dune/issues/530. The idea was indeed to change the meaning of `jobs` to the latter, although I haven't checked if it happened